modules/mig_with_percent/variables.tf

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "hostname" {
+  description = "Hostname prefix for instances"
+  default     = "default"
+}
+
+variable "region" {
+  description = "The GCP region where the managed instance group resides."
+}
+
+variable "instance_template_version0" {
```

Comment:
Are there more descriptive names for the two template groups rather than 0 and 1? Regardless, the names should be delimited like `version_0`.

Reply:
Changed to `initial_version` and `next_version`. Any ideas?